code/__defines/subsystems.dm

```diff
@@ -75,6 +75,7 @@ var/global/list/runlevel_flags = list(RUNLEVEL_LOBBY, RUNLEVEL_SETUP, RUNLEVEL_G
 // Subsystem fire priority, from lowest to highest priority
 // If the subsystem isn't listed here it's either DEFAULT or PROCESS (if it's a processing subsystem child)
 #define FIRE_PRIORITY_SHUTTLES		5
+#define FIRE_PRIORITY_SUPPLY		5
```

Comment:
This intentional to be both 5?

Reply:
There is no particular reason why they need to be different.  Many subsystems all share the default priority of 50.    
I set it as 5 because it is an *extremely* quick fire() proc, and it doesn't really matter if it gets delayed even several seconds since its SS_KEEP_TIMING.     I like multiples of 5 or 10 to be pretty, but it could be fine as 4.  Would that be less troubling?

Reply:
Got'cha. Bit unfamiliar with this type of code so whatever you think is fine.